test-network-function/platform/suite.go

```diff
@@ -57,68 +57,69 @@ import (
-		log.Infof("Test Configuration: %s", config)
-
-		containersUnderTest := common.CreateContainersUnderTest(config)
-		partnerContainers := common.CreatePartnerContainers(config)
-		fsDiffContainer := partnerContainers[config.FsDiffMasterContainer]
-		log.Info(containersUnderTest)
 
+		configData := common.ConfigurationData{}
+		configData.SetNeedsRefresh()
+		ginkgo.BeforeEach(func() {
+			common.ReloadConfiguration(&configData)
+		})
 		ginkgo.Context("Container does not have additional packages installed", func() {
 			// use this boolean to turn off tests that require OS packages
 			if !common.IsMinikube() {
-				if fsDiffContainer != nil {
-					for _, containerUnderTest := range containersUnderTest {
-						testFsDiff(fsDiffContainer.Oc, containerUnderTest.Oc)
-					}
-				} else {
-					log.Warn("no fs diff container is configured, cannot run fs diff test")
-				}
+				testPodsFsDiff(&configData)
 			}
 		})
 
 		testTainted()
 		testHugepages()
 
 		if !common.IsMinikube() {
-			for _, containersUnderTest := range containersUnderTest {
-				testBootParams(common.GetContext(), containersUnderTest.Oc.GetPodName(), containersUnderTest.Oc.GetPodNamespace(), containersUnderTest.Oc)
-			}
+			testBootParams(&configData)
 		}
 
 		if !common.IsMinikube() {
-			for _, containersUnderTest := range containersUnderTest {
-				// no test identifier defined, not an official test
-				testSysctlConfigs(common.GetContext(), containersUnderTest.Oc.GetPodName(), containersUnderTest.Oc.GetPodNamespace())
-			}
+			testSysctlConfigs(&configData)
 		}
 
 	}
 })
 
-// Helper to test that the PUT didn't install new packages after starting, and report through Ginkgo.
-func testFsDiff(masterPodOc, targetPodOc *interactive.Oc) {
-	ginkgo.It(fmt.Sprintf("%s(%s) should not install new packages after starting", targetPodOc.GetPodName(), targetPodOc.GetPodContainerName()), func() {
-		defer results.RecordResult(identifiers.TestUnalteredBaseImageIdentifier)
-		targetPodOc.GetExpecter()
-		containerIDTester := containerid.NewContainerID(common.DefaultTimeout)
-		test, err := tnf.NewTest(targetPodOc.GetExpecter(), containerIDTester, []reel.Handler{containerIDTester}, targetPodOc.GetErrorChannel())
-		gomega.Expect(err).To(gomega.BeNil())
-		testResult, err := test.Run()
-		gomega.Expect(testResult).To(gomega.Equal(tnf.SUCCESS))
-		gomega.Expect(err).To(gomega.BeNil())
-		containerID := containerIDTester.GetID()
-
-		fsDiffTester := cnffsdiff.NewFsDiff(common.DefaultTimeout, containerID)
-		test, err = tnf.NewTest(masterPodOc.GetExpecter(), fsDiffTester, []reel.Handler{fsDiffTester}, masterPodOc.GetErrorChannel())
-		gomega.Expect(err).To(gomega.BeNil())
-		testResult, err = test.Run()
-		gomega.Expect(testResult).To(gomega.Equal(tnf.SUCCESS))
-		gomega.Expect(err).To(gomega.BeNil())
+// testPodsFsDiff test that all PUT didn't install new packages are starting
+func testPodsFsDiff(configData *common.ConfigurationData) {
+	ginkgo.It("platform-fsdiff", func() {
```

Comment:
You sure we want to remove the existing text? I am ok with adding the prefix.

Reply:
Yes. I can have another story just to go through naming once and for all.. and establish some standard.